tests/Unit/InventoryServiceTest.php

```
@@ -0,0 +1,348 @@
+function mockFilesystem(bool $exists = true, string $yamlContent = '', bool $throwOnMkdir = false, bool $throwOnDump = false): Filesystem
+{
+    return new class ($exists, $yamlContent, $throwOnMkdir, $throwOnDump) extends Filesystem {
+        public function __construct(
+            private bool $exists,
+            private string $yamlContent,
+            private bool $throwOnMkdir,
+            private bool $throwOnDump
+        ) {
+        }
+
+        public function exists($files): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$files'. (undefined)

(UnusedFormalParameter)